input/pagecontent/StructureDefinition-ViewDefinition-notes.md

```diff
@@ -133,102 +136,55 @@ like this:
+|Pre-Process to Create or Resolve Keys|The most difficult case is systems where the resource `id` is a not a reliable row key, and resources have multiple `identifier`s with no clear way to select one for the key.<br><br>In this case, implementations will likely have to fall back to some form of preprocessing to determine appropriate keys. This may be accomplished by:<br><br><ul><li>Pre-processing all data to have clear resource `id`s or "primary" `identifier`s and using one of the options above.</li><li>Building some form of cross-link table dynamically within the implementation itself based on the underlying data. For instance, if an implementation's [getResourceKey()](#getresourcekey--keytype) uses a specific identifier system, [getReferenceKey()](#getreferencekeyresource-resourcetypecode--keytype) could use use a pre-built cross-link table to find the appropriate identifier-based key to return.</li></ul>
 
 There are many variations and alternatives to the above. This spec simply asserts that implementations must
 be able to produce a row key for each resource and a matching key for references pointing at that resource,
 and intentionally leaves the specific mechanism to the implementation.
 
-### Contained resources
-The current version of this specification requires compliant implementations to extract contained resources
-needed for view definitions into independent resources, which can then be accessed via 
-[getReferenceKey()](#getreferencekeyresource-resourcetypecode--keytype) like any other resource. Implementations SHOULD 
-normalize these resources appropriately whenever possible, such as by eliminating duplicate resources contained in many parents.
-
-Contained resources have different semantics than other FHIR resources since they don't have an independent identity,
-and the same logical record may be duplicated across many containing resources. This makes good SQL practices difficult since the 
-data is denormalized and ambiguous. For instance, `Patient.generalPractitioner` may be a contained resource that may or may not be
-the same practitioner seen in other `Patient` resources. Therefore this spec currently requires system pre-process such
-data into normalized, independent resources if needed.
+### Contained Resources
+This specification requires implementators to extract contained resources,
```

Comment:
Suggest "implementers" instead of "implementators".

Reply:
Fixed!